# ChemSensor.msg
std_msgs/Header header
TemperatureSensor temp
HumiditySensor rh
AnalogSensor s1
AnalogSensor s2
AnalogSensor s3
